x-pack/plugin/esql/compute/build.gradle

```diff
@@ -54,11 +55,11 @@ def prop(Type, type, TYPE, BYTES, Array, Hash) {
 }
 
 tasks.named('stringTemplates').configure {
-  var intProperties      = prop("Int", "int", "INT", "Integer.BYTES", "IntArray", "LongHash")
-  var longProperties     = prop("Long", "long", "LONG", "Long.BYTES", "LongArray", "LongHash")
-  var doubleProperties   = prop("Double", "double", "DOUBLE", "Double.BYTES", "DoubleArray", "LongHash")
-  var bytesRefProperties = prop("BytesRef", "BytesRef", "BYTES_REF", "org.apache.lucene.util.RamUsageEstimator.NUM_BYTES_OBJECT_REF", "", "BytesRefHash")
-  var booleanProperties  = prop("Boolean", "boolean", "BOOLEAN", "Byte.BYTES", "BitArray", "")
+  var intProperties      = prop("Int", "int", "Integer", "INT", "Integer.BYTES", "IntArray", "LongHash")
+  var longProperties     = prop("Long", "long", "Long", "LONG", "Long.BYTES", "LongArray", "LongHash")
+  var doubleProperties   = prop("Double", "double", "Double", "DOUBLE", "Double.BYTES", "DoubleArray", "LongHash")
+  var bytesRefProperties = prop("BytesRef", "BytesRef", "", "BYTES_REF", "org.apache.lucene.util.RamUsageEstimator.NUM_BYTES_OBJECT_REF", "", "BytesRefHash")
```

Comment:
Here I added a "Wrapper" value. Because Int != Integer (It's the only case btw). So instead of adding a condition, added this here. If you think it's an overuse of those properties, I can inline the condition instead

Reply:
Nah, you are good. $Wrapper$ is clearer than ``$if(blhablbalha)())()`